pages/scripts/scripts.js

```diff
@@ -199,36 +199,59 @@ function getPageName(sectionName) {
   return pageSectionParts.filter((subPath) => subPath !== '').join(':');
 }
 
-export function clearDataLayer() {
-  window.adobeDataLayer = [];
+/*
+ * Sanitizes a name for use as a js property name.
+ * @param {string} name The unsanitized name
+ * @returns {string} The camelCased name
+ */
+export function toCamelCase(name) {
+  return toClassName(name).replace(/-([a-z])/g, (g) => g[1].toUpperCase());
 }
 
-export function pushOneTrustConsentGroups() {
-  window.adobeDataLayer = window.adobeDataLayer || [];
-  const dl = window.adobeDataLayer;
-  dl.push({
-    event: 'LaunchOTLoaded',
-    // eslint-disable-next-line no-undef
-    OnetrustActiveGroups: typeof OnetrustActiveGroups !== 'undefined' ? OnetrustActiveGroups : '',
-  });
+export async function fetchPlaceholders(prefix = 'default') {
```

Comment:
Why was this added to scripts?

This is already in lib-franklin, we can import it from there.

Reply:
oh, in pga-mitsubishi-electric-championship, it was added in the scripts file. I will import it from lib-franklin